src/js/construct-element.js

```diff
@@ -1,133 +1,72 @@
-import { buildActions, throwError } from './helpers';
-
 export default {
 	/**
-	* Build a full alert message element. Used when there is no message element in the DOM.
-	* @returns {HTMLElement} Returns the new alert type message element
+	* Build a full message element. Used when there is no message element in the DOM.
+	* @returns {HTMLElement} Returns the new message element
 	*/
-	alertMessage: (opts) => {
-		const alertMessageEl = document.createElement('div');
-		alertMessageEl.classList.add(opts.messageClass, opts.typeClass, `${opts.messageClass}--closed`);
+	message: (opts) => {
+		const messageElement = document.createElement('div');
+		messageElement.classList.add('o-message', `o-message--${opts.type}`, 'o-message--closed');
```

Comment:
What if `opts.type` is null?

Reply:
it defaults to 'alert' if none is defined. Do you think it should not?